README.md

```diff
@@ -104,6 +104,31 @@ docker run -d -p [A host port]:80 [Image name]:[Image tag]
 
 3. Open your favorite browser at ```localhost:[Chosen host port]``` and enjoy ! :whale:
 
+# Running SonarQube
+
+## By Script : 
+
+1. Run Sonar in container : ```docker-compose -f ./docker/sonar.yml up -d```
+   
+2. Wait container was up Run ```SonarAnalysis.ps1``` and go to http://localhost:9001
+
+## Manually : 
+
+1. Run Sonar in container : ```docker-compose -f ./docker/sonar.yml up -d```
+
+2. Install sonar scanner for .net :
+   
+ ```dotnet tool install --global dotnet-sonarscanner```
+
+3. Run ```dotnet sonarscanner begin /d:sonar.login=admin /d:sonar.password=admin /k:"AwesomeKey" /d:sonar.host.url="http://localhost:9001"```
```

Comment:
can we unify the way sonar is triggered either by container or manually? Use the ps1 script in both places ? or in none? having a cli to copy/paste from the Readme.md is good enough

Reply:
In the two cases i run the docker and the ps1 script only launches the 4 commands that can be seen in the manual section